docs/CLI/admin/groups.md

```diff
@@ -0,0 +1,196 @@
+
+# <ADMIN> Primehub Groups
+
+```
+Usage: 
+  primehub admin groups <command>
+
+Manage groups
+
+Available Commands:
+  create               Create a group
+  delete               Delete the group by id
+  get                  Get the group info by id
+  list                 List groups
+  update               Update the group
+
+Options:
+  -h, --help           Show the help
+
+Global Options:
+  --config CONFIG      Change the path of the config file (Default: ~/.primehub/config.json)
+  --endpoint ENDPOINT  Override the GraphQL API endpoint
+  --token TOKEN        Override the API Token
+  --group GROUP        Override the current group
+  --json               Output the json format (output human-friendly format by default)
+
+```
+
+
+### create
+
+Create a group
+
+
+```
+primehub admin groups create
+```
+ 
+
+* *(optional)* file: the file path of the configurations
+
+
+
+
+### delete
+
+Delete the group by id
+
+
+```
+primehub admin groups delete <id>
+```
+
+* id: the group id
+ 
+
+
+
+
+### get
+
+Get the group info by id
+
+
+```
+primehub admin groups get <id>
+```
+
+* id: the group id
+ 
+
+
+
+
+### list
+
+List groups
+
+
+```
+primehub admin groups list
+```
+ 
+
+* *(optional)* page: the page of all data
+
+
+
+
+### update
+
+Update the group
+
+
+```
+primehub admin groups update <id>
+```
+
+* id: the group id
+ 
+
+* *(optional)* file: the file path of the configurations
+
+
+
+ 
+
+## Examples
+
+### Fields for creating or updating
+
+| field | required | type | description |
+| --- | --- | --- | --- |
+| name | required | string | must start with a letter or numeric, '-' and '_' are allowed, and the length should be more than 2. |
+| displayName | optional | string | display name |
+| enabledDeployment | optional | boolean | groups with enabled model deployment are able to deploy/serve models |
+| maxDeploy | optional | int | limit on the amount of deployments for this group |
+| enabledSharedVolume | optional | boolean | the shared volume is shared among members in the group. |
+| sharedVolumeCapacity | optional | int | the capacity of the shared volume in GB |
+| launchGroupOnly | optional | boolean | if the shared volume is only mounted in the launch group |
```

Comment:
We recommend removing all `launchGroupOnly` from this table and all examples.
because `launchGroupOnly` should be retired in the future and not recommended to use anymore.

Reply:
ok, will remove this field on both doc and notebook.